examples/bri-3/prisma/seed.ts

```diff
@@ -23,17 +24,33 @@ async function main() {
 
   await prisma.bpiSubject.create({
     data: {
-        name: 'BpiAdmin',
-        description: 'Internal Bpi Subject of this Bpi',
-        publicKey: '0x044e851fa6118d0d33f11ebf8d4cae2a25dca959f06c1ab87b8fec9ccbf0ca0021b7efc27c786f9480f9f11cfe8df1ae991329654308611148a35a2277ba5909fe',
-        // private key 0x0fbdb56ab0fecb2f406fa807d9e6558baedacc1c15c0e2703b77d4c08441e4fe, used for testing purposes only
-        loginNonce: '',
-        roles: {
-          connect: {
-            id: internalBpiSubjectRole.id
-          },
+      name: 'BpiAdmin',
+      description: 'Internal Bpi Subject of this Bpi',
+      publicKeys: {
+        createMany: {
+          data: [
+            {
+              type: PublicKeyType.ECDSA,
+              value:
+                '0x044e851fa6118d0d33f11ebf8d4cae2a25dca959f06c1ab87b8fec9ccbf0ca0021b7efc27c786f9480f9f11cfe8df1ae991329654308611148a35a2277ba5909fe',
```

Comment:
these two are the same? This is probably not intended as they are of different type?

Reply:
Done.